### PR TITLE
PLT-3359 EE: Add the ability to restrict the user roles that can send team invites

### PR DESCRIPTION
**Pull request by jwilander**
No corresponding enterprise PR.

### CODE REVIEW

File: webapp/components/admin_console/localization_settings.jsx [hunk 1]
```diff
@@ -49,23 +49,16 @@ export default class LocalizationSettings extends AdminSettings {
         return (
             <h3>
                 <FormattedMessage
-                    id='admin.general.title'
-                    defaultMessage='General Settings'
+                    id='admin.general.localization'
```
Comment (enahum) [inline webapp/components/admin_console/localization_settings.jsx]: renamed here but not on `en.json`

**Comment by enahum** [comment]
+1 othen than the localization thing

**Comment by coreyhulen** [comment]
+1

File: webapp/components/admin_console/localization_settings.jsx (hunk 1 above)
Comment (jwilander) [inline webapp/components/admin_console/localization_settings.jsx]: It's not actually renamed, I just used a different but existing translation. I didn't delete the old one though so doing that..

**Comment by philku79** [comment]
👍

**Comment by esethna** [comment]
Thanks @jwilander, found an issue during testing:

1. For all settings of "Enable sending team invites from", the Invite New Members dialog is visible in the Main Menu but it is disabled even though email sign up is turned on in the console.

**Comment by jwilander** [comment]
You must have email configured for it to be enabled, otherwise the emails won't go anywhere. I didn't change that it's how it has always worked. Propose a different ticket to fix that if we want to change it?

**Comment by esethna** [comment]
@jwilander, I'll bring this up in UX to see if we want to open a ticket on this. I will definitely update the help text on the "Send Email Notification" field to make this more obvious.

**Comment by esethna** [comment]
:+1: otherwise, tested and works as [speced](https://mattermost.atlassian.net/browse/PLT-3359)

**Comment by jwilander** [comment]
Thanks @esethna !

**Pull request merged**